src/globalStyles.js

```diff
@@ -15,29 +19,33 @@
 
 @font-face {
   font-family: "Roboto";
-  src: url("./fonts/Roboto/Roboto-Regular.ttf");
+  src: local("robotoRegular"),
+  url(${robotoRegular});
```

Comment:
Don`t you need to add the format here as well?

```suggestion
  url(${robotoRegular}) format('truetype');
```

Reply:
Thanks!